src/main/java/Demo.java

```diff
@@ -1,7 +1,100 @@
 package main.java;
 
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Demo {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        calculator1();
+    }
+
+    public static void calculator1() {
+        Scanner sc = new Scanner(System.in);
+        String inputString;
+        System.out.println("Enter a string :");
+        System.out.println("Trying to push directly to git.");
+        inputString = sc.nextLine();
+
```

Comment:
Why are you using two blank lines instead of one?

Reply:
A blank line is often a good indicator that this should be split into a function. Effectively, this is a new block of code.